lib/js/src/streaming/video/VideoStreamingParameters.js

```diff
@@ -61,14 +61,14 @@ class VideoStreamingParameters {
         this._bitrate = bitrate || DEFAULT_BITRATE;
         this._interval = interval || DEFAULT_INTERVAL;
 
-        if (!resolution) {
+        if (resolution === null) {
             resolution = new ImageResolution();
             resolution.setResolutionWidth(DEFAULT_WIDTH);
             resolution.setResolutionHeight(DEFAULT_HEIGHT);
         }
         this._resolution = resolution;
 
-        if (!format) {
+        if (format === null) {
             this._format = new VideoStreamingFormat();
             this._format.setProtocol(DEFAULT_PROTOCOL);
             this._format.setCodec(DEFAULT_CODEC);
```

Comment:
Operating on the wrong variable. Will be overwritten with null `format` on line 77 below.